pymc/model.py

```diff
@@ -1765,8 +1765,8 @@ def set_data(new_data, model=None):
     model = modelcontext(model)
 
     for variable_name, new_value in new_data.items():
-        model.set_data(variable_name, new_value)
-
+        model.set_data(variable_name, new_value,coords)
+    """return model.set_data(new_data, coords)"""
```

Comment:
This line should be removed. This function modifies the data stored in the model class and does not have and should not have any return function. If you look at the code of pymc.Model.set_data you'll see that it doesn't have any return value either. 

Adding this as a string makes the code run because this line is now ignored instead of being executed, so it should be removed to avoid adding code that is never executed.

Reply:
I would definitely do the required changes and keep in mind what ever you have said